### PR TITLE
feature: Not connected alert banner in React

### PR DESCRIPTION
**Pull request by yomybaby**
### TL;DR
Added a network status banner to the main layout component.

| before | after |
| -- | -- |
|  |   | 

### What changed?
- Introduced `NetworkStatusBanner` component.
  - Displays when the user is offline
- Updated translations for the offline message in multiple languages.
- Removed old snackbar-based offline indicator.

### How to test?
1. Go offline and verify that the new banner appears with the appropriate message.

### Why make this change?
Enhances user experience by providing clearer visual feedback on network status directly within the main layout.

---

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by yomybaby** [comment]
* **#2493** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2493?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2492** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2492?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yomybaby and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/NetworkStatusBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    border-bottom: 1px solid ${token.colorWarningBorder} !important;
+  `,
+}));
+const NetworkStatusBanner = () => {
+  const { t } = useTranslation();
+  const network = useNetwork();
+
+  const { styles } = useStyles();
+
+  const [softTimeout, setSoftTimeout] = useAtom(networkSoftTimeoutAtom);
```
Comment (yomybaby) [inline react/src/components/NetworkStatusBanner.tsx:23]: FYI, `softTimeot` feature is handled by #2493 .

**Review by ironAiken2 — CHANGES_REQUESTED** [review]
After the network is reconnected, the notifiaction will output a network shutdown statement. How about removing that wording, or modifying it to appear at the same time as the banner?

**Comment by yomybaby** [comment]
@ironAiken2 I created a new stack to improve notifications. Please check that #2502 PR and resolve a current PR.

**Review by ironAiken2 — APPROVED** [review]
LGTM!

**Pull request merged**